.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - "7.2"
```

Comment:
PHPUnit 9.5 has a hard requirement on PHP 7.3 - we should remove 7.2 from the runtimes we test against if it is being upgraded